freelist.go

```diff
@@ -171,7 +174,11 @@ func (f *freelist) read(p *page) {
 	copy(f.ids, ids)
 
 	// Make sure they're sorted.
-	sort.Sort(pgids(f.ids))
+	for i, v := range f.ids[:len(f.ids)-1] {
```

Comment:
@neurodrone pgid is just an integer, so that it's copy is very very cheap. And this custom loop will use direct integer comparisons.

sort.IsSorted will be much slower, cause it will call .Less on every iteration through interface description, and function call is many times slower than not doing function call.